libs/langchain-google-common/src/utils/media_core.ts

```diff
@@ -0,0 +1,465 @@
+import { v4 as uuidv4 } from "uuid";
+import { BaseStore } from "@langchain/core/stores";
+import { Serializable } from "@langchain/core/load/serializable";
+
+export interface MediaBlobParameters {
+  data?: Blob;
+
+  metadata?: Record<string, unknown>;
+
+  path?: string;
+}
+
+/**
+ * Represents a chunk of data that can be identified by the path where the
+ * data is (or will be) located, along with optional metadata about the data.
+ */
+export class MediaBlob
```

Comment:
CC @eyurtsev

Reply:
I want to make clear for @eyurtsev's benefit that the name chosen is `MediaBlob` for two reasons:
1. To distinguish it from a `Blob`, which is already a major class in JS/TS
2. To easily and clearly use a JS/TS `Blob` as a property in this object.

But the goal was to have the same interface structure of a LangChain Python `Blob`.